lib/phoenix_live_view.ex

```diff
@@ -508,6 +509,19 @@ defmodule Phoenix.LiveView do
                   "got: #{inspect(other)}"
       end
 
+    log =
+      case Keyword.get(opts, :log, :info) do
+        level when is_atom(level) ->
+          level
+
+        false ->
+          false
+
+        other ->
+          raise ArgumentError,
+                ":log expects an atom or false, got: #{inspect(other)}"
+      end
```

Comment:
Since both `true` and `false` are atoms and seems like `true` isn't a valid value, what about changing the check to:

```suggestion
    level = Keyword.get(opts, :log, :info)
    
    log =
      if not is_atom(level) or level == true do
        raise ArgumentError, ":log expects an atom or false, got: #{inspect(level)}"
      else
        level
      end
```